app/schema.graphql

```diff
@@ -113,6 +153,16 @@ type EvaluationRunConfig implements RunConfig {
   method: String
 }
 
+type FloatAggregation implements Aggregation {
```

Comment:
Are these really aggregations or stats or something in between? What’s the overlap with actual fo aggregations?

Reply:
They are something in between. We can rename them now or later, if appropriate. I think introducing the real aggregation API into the GraphQL schema is out of scope.

I'm introducing a new field for `/aggregation` in GraphQL, but it will likely just accept and return JSON without typing.

Reply:
The goal for me was to add typing to the statistics data required by the App via GraphQL and change the request to be path specific. Developing with statistics is much more straightforward now.